src/main/java/uk/gov/hmcts/divorce/ciccase/model/RepresentativeCIC.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.divorce.ciccase.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasLabel;
+
+@Getter
+@AllArgsConstructor
+public enum RepresentativeCIC implements HasLabel {
+    @JsonProperty("RepresentativeCIC")
+    REPRESENTATIVE("Representative");
+    private final String label;
+    public boolean isRepresentativeCIC() {
+        return REPRESENTATIVE.name().equalsIgnoreCase(this.name());
+    }
```

Comment:
If REPRESENTATIVE is an enum, use the == operator to check equality, not String equality.
== is null safe, .equals is not (Though you're not going to get an NPE here, it's just better practice)